web/event_handler_test.go

```diff
@@ -20,7 +20,7 @@ func TestWebHandler_DropUnknownEventType(t *testing.T) {
 	t.Parallel()
 
 	// given
-	queue := make(chan event)
+	queue := make(chan event, 10)
```

Comment:
might not make sense anymore, but it helped to pass any tests at the beginning. will change